content/community/moderation.md

```diff
@@ -62,10 +62,13 @@ Moderators need to stick to the [Codewars Code of Conduct](/community/rules/), t
 
 They can be recognized by the symbol of a shield (🛡️) displayed next to their username.
```

Comment:
Took the SVG from CW. Will see if it works, did test in the devtools...
```suggestion
They can be recognized by the symbol of a shield (<svg height="20" fill="none" style="display: inline-block" stroke="currentColor" stroke-width="2" viewBox="0 0 24 24"><path d="M12 2a12 12 0 0 0 8.5 3a12 12 0 0 1 -8.5 15a12 12 0 0 1 -8.5 -15a12 12 0 0 0 8.5 -3"></path></svg>) displayed next to their username.
```

Reply:
for kazk:

`<svg class="inline-block" fill="currentColor" height="15" stroke="currentColor" stroke-width="1" viewBox="0 0 24 24" width="15"><path d="M12 4l4 6l5 -4l-2 10h-14l-2 -10l5 4z"></path></svg>`

Reply:
Indeed, I forgot that this was made with Tailwind...

Reply:
I will revert it for now, because I am not really sure of some things. I wonder if absolute sizing or putting SVG inline are good ideas, if styling/theming will be handled correctly, etc.

If you have the crown SVG too, then guys create separate PR and we will see both icons in the preview.